web/packages/teleport/src/Discover/Database/EnrollRdsDatabase/RdsDatabaseList.tsx

```diff
@@ -211,3 +221,25 @@ function labelMatcher(
     });
   }
 }
+
+const Cell: React.FC<{ disabled: boolean; width?: string }> = ({
+  disabled,
+  width,
+  children,
+}) => {
+  return (
+    <TableCell
+      width={width}
+      title={
+        disabled
+          ? 'this RDS database is already enrolled and is a part of this cluster'
+          : null
+      }
+      css={`
+        opacity: ${disabled ? '0.5' : '1'};
```

Comment:
i think it would require more work to try to `color` (with the disabled text color) the elements like status color and label pills. But is using opacity wrong here? Wouldn't it be theme independent? (like theme colors wouldn't affect other than to reduce it's transparency)

<img width="736" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/31e6f50d-2991-4051-8828-b9661b7f6678">
<img width="1240" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/897b242d-9448-471f-8f14-b415d48d3d95">

Reply:
Ok, I wasn't sure about the custom 0.5 value (our default opacity is 0.36), so I thought about using the value from theme, but as you said, it specifies the entire color, so it will be more difficult to implement. I think we can leave it as is.